public/spec/auth/spec.md

```diff
@@ -7,7 +7,7 @@ _Updated_: 2022/02/17
 _Edition_: Swan Lake  
 
 ## Introduction
-This is the specification for the Auth standard library of [Ballerina language](https://ballerina.io/), which is used for authorization of listeners and clients (HTTP, gRPC, GraphQL, WebSocket, WebSub, etc.).
+This is the specification for the Auth Ballerina library of [Ballerina language](https://ballerina.io/), which is used for authorization of listeners and clients (HTTP, gRPC, GraphQL, WebSocket, WebSub, etc.).
```

Comment:
It is incorrect to say Library for single package. 

Package is the sharable unit and library is a collection of such packages. So it is incorrect to say `Auth Ballerina library`. So instead something like `Auth package of Ballerina Library` or something will do.

@praneesha / @shafreenAnfar / @daneshk / @ThisaruGuruge

Reply:
Will be fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.